File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.java
// Last Updated: 2/5/2026, 9:22:02 AM
class Solution {
    public int maximumWealth(int[][] accounts) {
        int result=0;
        for ( int i=0 ; i<accounts.length ; i++){
            int sum=0;
            for (int j=0 ; j<accounts[i].length ; j++){
                sum+=accounts[i][j];
            }
            result = Math.max(result,sum);
        }
        return result;
    }
}

File: 1791-RichestCustomerWealth/1791-RichestCustomerWealth.py
# Last Updated: 2/5/2026, 9:22:09 AM
class Solution:
    def maximumWealth(self, accounts: List[List[int]]) -> int:
        result=0
        for i in range(len(accounts)):
            result=max(result,sum(accounts[i][::]))
        return result